rllib/policy/torch_policy.py

```diff
@@ -26,8 +27,15 @@ class TorchPolicy(Policy):
         dist_class (type): Torch action distribution class.
     """
 
-    def __init__(self, observation_space, action_space, config, model, loss,
-                 action_distribution_class):
+    def __init__(self,
+                 observation_space,
+                 action_space,
+                 config,
+                 model,
+                 loss,
+                 action_distribution_class,
+                 max_seq_len=20,
+                 get_batch_divisibility_req=None):
```

Comment:
Hmm what's the reason it needs to be a lambda instead of a concrete value like in TF?

Reply:
Not sure. It's currently a lambda in tf and eager as well.